docs/odh/oci-model-storage.md

```diff
@@ -0,0 +1,208 @@
+user and group, and also read permissions are granted to all users. This is
+important, because OpenShift runs containers with a random user ID and with the
+`root` group ID. The adjustment of the group and the privileges on the model files
+ensures that the model server can access them.
+
+Verify that the directory structure is good using the `tree` command:
+```shell
+tree
+
+.
+├── Containerfile
+└── models
+    └── 1
+        └── mobilenetv2-7.onnx
+```
+
+> [!NOTE]
+> Remember that the shown directory structure under `models/` is specific to OpenVINO.
+
+Create the OCI container image with podman, and upload it to a registry. For
+example, using Quay as the registry:
+```shell
+podman build --format=oci -t quay.io/<user_name>/<repository_name>:<tag_name> .
+podman push quay.io/<user_name>/<repository_name>:<tag_name>
+```
+
+> [!TIP]
+> When uploading your image, if your image repository is private, ensure you
+> are authenticated to the registry.
+
+## Deploying a model stored in an OCI image in a public repository
+
+Start by creating a namespace to deploy the model:
+```shell
+oc new-project oci-model-example
+```
+
+In the newly created namespace, you need to create a `ServingRuntime` resource
+configuring OpenVINO model server. The ODH project provides templates with
+configurations for some model servers, which you can list with the following
+command:
+```shell
+oc get template -n opendatahub
+
+NAME                                 DESCRIPTION                                                                        PARAMETERS    OBJECTS
+caikit-standalone-serving-template   Caikit is an AI toolkit that enables users to manage models through a set of...    0 (all set)   1
+caikit-tgis-serving-template         Caikit is an AI toolkit that enables users to manage models through a set of...    0 (all set)   1
+kserve-ovms                          OpenVino Model Serving Definition                                                  0 (all set)   1
+ovms                                 OpenVino Model Serving Definition                                                  0 (all set)   1
+tgis-grpc-serving-template           Text Generation Inference Server (TGIS) is a high performance inference engin...   0 (all set)   1
+vllm-runtime-template                vLLM is a high-throughput and memory-efficient inference and serving engine f...   0 (all set)   1
+```
+
+The template that is applicable for KServe and holds the OpenVINO configuration
+is the one named as `kserve-ovms`. To create an instance of it, run the
+following command:
+```shell
+oc process -n opendatahub -o yaml kserve-ovms | oc apply -f -
+```
+
+You can verify that the `ServingRuntime` has been created successfully with the
+following command:
+```shell
+oc get servingruntimes
+
+NAME          DISABLED   MODELTYPE     CONTAINERS         AGE
+kserve-ovms              openvino_ir   kserve-container   1m
+```
+
+Notice that the ServingRuntime has been created with `kserve-ovms` name.
```

Comment:
```suggestion
> [!NOTE]
> The ServingRuntime has been created with `kserve-ovms` name.
```

Reply:
Rather than being a side note, I wanted to imply that the user should "take note" of the name, as it is used on the `InferenceService`... So, not sure if it should be quoted as a `[!NOTE]`.

Reply:
my bad, I thought that it was `NOTE` not `Notice`